File: .sdk/tm/rb/utility/make_context.rb
# TempMailApiByBoomlify SDK utility: make_context
require_relative '../core/context'
module TempMailApiByBoomlifyUtilities
  MakeContext = ->(ctxmap, basectx) {
    TempMailApiByBoomlifyContext.new(ctxmap, basectx)
  }
end

File: py/tempmailapibyboomlify_sdk/utility/make_context.py
# TempMailApiByBoomlify SDK utility: make_context

from tempmailapibyboomlify_sdk.core.context import TempMailApiByBoomlifyContext


def make_context_util(ctxmap, basectx):
    return TempMailApiByBoomlifyContext(ctxmap, basectx)
